src/styles/theme.css

```
@@ -503,4 +503,7 @@
 	--color-warning-8: #836000;
 	--color-warning-9: #644A00;
 	--color-warning: #EFAF00;
+	--font-Lato: Lato;
```

Stylelint (17.6.0)

[error] Expected "Lato" to be "lato" (value-keyword-case)

(value-keyword-case)